ote/src/cljs/ote/communication.cljs

```diff
@@ -4,32 +4,42 @@
             [cognitect.transit :as t]
             [ote.transit :as transit]))
 
+(defonce base-url (atom ""))
+
+(defn set-base-url! [url]
+  (reset! base-url url))
+
 (defn transit-request-format []
   (ajax/transit-request-format transit/write-options))
 
 (defn transit-response-format []
   (ajax/transit-response-format {:reader (t/reader :json transit/read-options)
                                  :raw true}))
 
+(defn- request-url [url]
+  (str @base-url url))
+
 (defn get!
   "Make a GET request to the given URL.
   URL parameters can be given with the `:params` key.
   Callbacks for successfull and failure are provided with `:on-success` and `:on-failure`
   keys respectively"
   [url {:keys [params on-success on-failure response-format]}]
-  (GET url {:params params
-            :handler on-success
-            :error-handler on-failure
-            :response-format (or response-format (transit-response-format))}))
+  (GET (request-url url)
```

Comment:
What that request-url does to that url parameter? You can pass parameter straight to functions function? :)

Reply:
It prepends the base-url (if any). The relative path is different when we use OTE app embedded in CKAN.